plugins/snyk/api_token.go

```diff
@@ -39,7 +43,41 @@ var defaultEnvVarMapping = map[string]sdk.FieldName{
 	"SNYK_TOKEN": fieldname.Token,
 }
 
+type Config struct {
+	Token string `json:"api"`
+}
+
 func TrySnykConfigFile() sdk.Importer {
-	// TODO: Import `api` field from ~/.config/configstore/snyk.json
-	return importer.NoOp()
+	path := configFilePath()
+	return importer.TryFile(path, func(ctx context.Context, contents importer.FileContents, in sdk.ImportInput, out *sdk.ImportAttempt) {
```

Comment:
We don't support `XDG_CONFIG_HOME` in all the other importers either, so I think we should wait with adding support for Snyk until we have a more sustainable solution there. So for now just stick with `~/.config/configstore/snyk.json`.

Reply:
Ok, I'll get rid of `XDG_CONFIG_HOME` support here.